openvpn-formula/openvpn-formula.changes

```diff
@@ -0,0 +1,4 @@
+-------------------------------------------------------------------
+Mon Apr 27 14:35:26 UTC 2020 - Alexander Graul <agraul@suse.com>
+
+- Inital Package v0.1
```

Comment:
Typo here:

```suggestion
- Initial Package v0.1
```

Reply:
Good catch! Are typo fixes accepted after changelogs have been released?

Reply:
Yes, should not be a problem.